rotkehlchen/chain/ethereum/modules/curve/accountant.py

```diff
@@ -88,3 +36,7 @@ def event_settings(self, pot: 'AccountingPot') -> dict[str, TxEventSettings]:  #
                 accountant_cb=self._process_deposit_or_withdrawal,
             ),
         }
+
+    @property
+    def name(self) -> str:
```

Comment:
In the end why do this? Didn't we say that the counterparty is good enough or am I missing something?